compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -361,6 +361,20 @@ generateS390CompareAndBranchInstruction(TR::CodeGenerator * cg,
    return returnInstruction;
    }
 
+TR::Instruction *
+generateS390CompareAndBranchInstruction(TR::CodeGenerator * cg,
```

Comment:
@simonameng  Can you add some explanation about why we needed version of `generateS390CompareAndBranchInstruction` which takes `TR::RealRegister` as argument.